contracts/Proxy/BorrowerOperationsScript.sol

```diff
@@ -4,42 +4,88 @@ pragma solidity 0.6.11;
 
 import "../Dependencies/CheckContract.sol";
 import "../Interfaces/IBorrowerOperations.sol";
+import "../Dependencies/IERC20.sol";
 
 
-contract BorrowerOperationsScript is CheckContract {
+contract BorrowerOperationsScript {
     IBorrowerOperations immutable borrowerOperations;
+    address immutable LUSD;
 
-    constructor(IBorrowerOperations _borrowerOperations) public {
-        checkContract(address(_borrowerOperations));
+    constructor(IBorrowerOperations _borrowerOperations, address _LUSD) public {
         borrowerOperations = _borrowerOperations;
+        LUSD = _LUSD;
     }
 
-    function openTrove(uint _maxFee, uint _LUSDAmount, address _upperHint, address _lowerHint) external payable {
```

Comment:
Why all these format changes?

Reply:
Prettier...

Reply:
That’s quite subjective. Some of the changed lines are as short as 36 chars, so I don’t see the need for it. Besides, it wouldn’t be consistent with the rest of our contracts.

Reply:
Well that's right it doesn't match the style of the other contracts. will fix that.